src/cobra/core/gene.py

```diff
@@ -283,8 +336,18 @@ class GPR(Module):
     """
 
     def __init__(self, gpr_from: Union[Expression, Module, AST] = None, **kwargs):
+        """Initialize a gene.
+
+        Parameters
+        ----------
+        gpr_from: Expression, Module, AST
+            An AST expression that will be parsed to GPR.
+        **kwargs:
+            Further keyword arguments are passed on to the parent class.
+        """
         super().__init__(**kwargs)
         self._genes = set()
+        # noinspection PyTypeChecker
```

Comment:
What's the type error that you see here? Could try
```suggestion
        # noinspection PyTypeChecker
        self.body: Optional[whatever body is] = None
```

Reply:
Great suggestion. Thank you. That worked!